test/cpp/tensorexpr/test_block.cpp

```diff
@@ -0,0 +1,46 @@
+#include <sstream>
+#include <stdexcept>
+#include "test/cpp/tensorexpr/test_base.h"
+
+#include <cmath>
+
+#include "test/cpp/tensorexpr/padded_buffer.h"
+#include "torch/csrc/jit/tensorexpr/block_codegen.h"
+#include "torch/csrc/jit/tensorexpr/buffer.h"
+#include "torch/csrc/jit/tensorexpr/loopnest.h"
+#include "torch/csrc/jit/tensorexpr/tensor.h"
+
+#include <c10/util/Half.h>
+
+namespace torch {
+namespace jit {
+using namespace torch::jit::tensorexpr;
+
+static void testBlockTestVectorAdd_impl(int N, int block_size) {
```

Comment:
In addition to check BlockCodeGen can be constructed, it is better to check the generated code matches expectation to some extent.

Reply:
Same as above; I removed this outdated unit test for now. I may add a new one later.